requirements.txt

```diff
@@ -0,0 +1,3 @@
+overpy==0.3.1
+transitfeed==1.2.14
+osmapi==0.6.2
```

Comment:
Having a minimum version to start from is not a bad idea, because we never tested very old versions and they might fail miserable. Using `>=` at least starts with the latest known to work version.